x-pack/plugins/security_solution/public/explore/hosts/components/hosts_table/columns.tsx

```diff
@@ -163,5 +166,24 @@ export const getHostsColumns = (
     });
   }
 
+  if (isAssetCriticalityEnabled) {
+    columns.push({
+      field: 'node.criticality',
+      name: i18n.ASSET_CRITICALITY,
+      truncateText: false,
+      mobileOptions: { show: true },
+      sortable: false,
+      render: (assetCriticality: CriticalityLevelWithUnassigned) => {
+        if (!assetCriticality) return;
```

Comment:
In the events table, it renders `-` if no value in the field. Should we do the same here?
All hosts table in this branch:
<img width="2276" alt="Screenshot 2024-06-19 at 09 04 56" src="https://github.com/elastic/kibana/assets/6295984/4817d6b5-ccdd-4aa9-8aed-b198c15fa93a">

Events table:
<img width="2291" alt="Screenshot 2024-06-19 at 09 04 09" src="https://github.com/elastic/kibana/assets/6295984/051bceb6-9f0d-4e49-8224-b09783da7d33">

Reply:
good point 🤔 
I think it makes sense since we do the same even in the All Hosts table

![Screenshot 2024-06-19 at 10 36 13](https://github.com/elastic/kibana/assets/2423976/7646be1c-e45f-4824-a4d6-bc7f1b0bce22)

I'll update.

Reply:
Done

Reply:
Thanks for doing that!